### ISSUE TITLE
Displaying in extensions. 0.92.3 Not opening box

### ISSUE THREAD

**Issue by Haleybelinda** [Neon22/inkscape-LasercutBox#13]
Hi after much effort I finally got this extension to display in my extensions drop down ( I havent managed the jigsaw.) However, when I choose lasercut box, nothing happens. no box appears for me to insert dimensions. Do I need a later inkscape? or is it something else please?

**Comment by Neon22** [comment]
Inkscape is now up to version 1.3. I would suggest you update - there is a lot of good stuff and many fixes to units etc.
Is there a reason you need such an old version

**Comment by Neon22** [comment]
can confirm its working in 1.3

**Comment by Haleybelinda** [comment]
I did indeed upgrade to 1.3 and laserbox and laser jigsaw are there. However, I don’t seem to have the conical box or scale or the quickjoint in the 1.3. Do I need to do something? I do see the conical box but it’s greyed out. Quick joint and scale have disappeared 🤔 thanks for your help

**Comment by Neon22** [comment]
scale was broken. I think moini was working on it. no idea on her progress. Check the extensions link at inkscape for what's available in 1.3 - sorry for lack of detail - been working on another project for some time and not up to speed on 1.3 at the moment

**Comment by Haleybelinda** [comment]
Thank you, Neon.
I wanted it for possible rulers if you have any suggestions. I will go and browse. Do you happen to know if the quick joint is still working as I seem to have lost that too?
Thank you very much in advance for your help
Haley

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Neon22** [comment]
yes scale was good but only worked in a small subset of cases so Moini pulled it. The code should work with minor adj if you want to try converting it. Or contact Moini.
In fact any extensions should be able to be migrated if you can find the original source. If you can find the original sources I might be able to help you but maybe not...

**Comment by Haleybelinda** [comment]
Thank you so much
